core/camel-core-model/src/main/java/org/apache/camel/builder/BuilderSupport.java

```diff
@@ -453,6 +453,10 @@ public void setContext(CamelContext context) {
         this.context = context;
     }
 
+    public boolean hasErrorHandlerBuilder(){
```

Comment:
there may be a checkstyle issue here (no space before bracket)

Reply:
Yep, did not notice that, thanks!